tasks/broad/RNAWithUMIsTasks.wdl

```diff
@@ -600,24 +606,31 @@ task GroupByUMIs {
 
     String docker = "us.gcr.io/broad-gotc-prod/umi_tools:1.0.0-1.1.1-1638821470"
     Int cpu = 2
-    Int memory_mb = 7500
+    # Int memory_mb = ceil(1.2 * size([bam, bam_index], "GiB")) + 4096 # Using the empirically estimated slope of (max memory usage) as a function of (input disk size)
```

Comment:
Is this code still needed? If not please clean up

Reply:
cleaned up